www/src/pages/editor/EditorPage.tsx

```diff
@@ -1,15 +1,33 @@
-import { Flex } from "antd";
+import { Button, Flex, Space } from "antd";
 import { observer } from "mobx-react-lite";
 import { SearchBar, useSearchArgs } from "./SearchBar";
 import { DedupMineralSiteTable } from "./DedupMineralSiteTable";
+import { useState } from "react";
+import { NewMineralSiteModal } from "./NewMineralSiteModal";
+import { Commodity } from "models";
+import { PlusOutlined } from "@ant-design/icons";
 
 export const EditorPage = observer(() => {
   const [searchArgs, normSearchArgs, setSearchArgs] = useSearchArgs();
+  const [isModalVisible, setIsModalVisible] = useState(false);
+
+  const handleOpenModal = () => setIsModalVisible(true);
+  const handleCloseModal = () => setIsModalVisible(false);
 
   return (
     <Flex vertical={true} gap="small">
       <SearchBar searchArgs={searchArgs} setSearchArgs={setSearchArgs} />
+      <Space style={{ marginBottom: 16 }}>
```

Comment:
does this create a separate row for the add button?

Reply:
yes

Reply:
it should be in a single line